SingularityService/src/main/java/com/hubspot/singularity/data/UsageManager.java

```diff
@@ -123,6 +140,10 @@ public SingularityCreateResult saveSpecificTaskUsage(String taskId, SingularityT
     return save(getSpecificTaskUsagePath(taskId, usage.getTimestamp()), usage, taskUsageTranscoder);
   }
 
+  public SingularityCreateResult saveSpecificClusterUtilization(SingularityClusterUtilization utilization) {
+    return save(getSpecificClusterUtilizationPath(utilization.getTimestamp()) , utilization, clusterUtilizationTranscoder);
```

Comment:
is there any point at which we would want a history of these? If we don't need a history of summaries, we might as well `save` the data to the summary path (without timestamp) and just overwrite when there is new data.